aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+      These regions are not always updated as a result of a user interaction.
+      By marking such elements as live regions, users of assistive technology can be informed of the updated content automatically.
+    </p>
+
+    <p>
+      Examples of live regions are a chat log and an error message.
+      However, different kinds of live regions have different expected behavior;
+      users might expect an error message to disrupt what they are doing, since the error is important,
+      but want to wait with reading new chat messages until they are done typing their own message.
+    </p>
+
+    <section id="live_region_states_and_properties">
+      <h3>Live Region States and Properties</h3>
+
+      <p>
+        ARIA has the following attributes mark up live regions.
+      </p>
+
+      <table>
+        <thead>
+        <tr>
+        <th>State/property</th>
+        <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><code>aria-live</code></td>
+            <td>Enables a live region.</td>
+           </tr>
+          <tr>
+            <td><code>aria-atomic</code></td>
+            <td>Indicates what content to announce.</td>
+          </tr>
+          <tr>
+            <td><code>aria-relevant</code></td>
+            <td>Indicates which content changes are relevant.</td>
+          </tr>
+          <tr>
+            <td><code>aria-busy</code></td>
+            <td>Defers content updates. Not specific to live regions. See the <a href="#aria-busy">next section</a>.</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <section id="enable_live_regions_with_aria-live">
+        <h4>Enable Live Regions with <code>aria-live</code></h4>
+
+        <p>
+          The <code>aria-live</code> attribute indicates that an element is a live region.
+          Some roles implicitly set <code>aria-live</code>; this is discussed in a later section.
+        </p>
+
+        <ul>
+          <li><code>assertive</code>: assistive technologies can interrupt the user to provide this information,
+          but will not move the current focus, so the user&#39;s work flow is not interrupted.</li>
+
+          <li><code>polite</code>: assistive technologies will provide this information after the user is done with their current task.</li>
+
+          <li><code>off</code>: assistive technologies will not provide this information unless this region has focus.</li>
+        </ul>
+
+        <p>
+          With the exception of a few roles, the default value of <code>aria-live</code> is <code>off</code>.
+        </p>
+
+        <p>
+          For example, when the user types into a search field, the page could update the page with search results as the user is typing.
+          To inform users of assistive technology that this has happened, a polite live region can be used.
```

Comment:
```suggestion
          To inform assistive technology users that this has happened, a polite live region can be used.
```

Remove potential confusion over "To inform users _of something that happened_..."